bedrock/newsletter/templates/newsletter/includes/form.html

```diff
@@ -54,13 +54,7 @@ <h4 class="mzp-c-newsletter-subtitle">{{ subtitle }}</h4>
       </div>
       {% endif %}
 
-      <label for="id_email">
-      {% if email_label %}
-        {{ email_label }}
-      {% else %}
-        {{ ftl('newsletter-form-your-email-address') }}
-      {% endif %}
-      </label>
+      {{ form.email.label_tag(ftl('newsletter-form-your-email-address') if not email_label else email_label) }}
```

Comment:
**issue (potentially blocking)**: based on this condition I would expect to see an email label on the developer form at bottom of page: http://localhost:8000/en-US/firefox/developer/ (not a regression as the live version does not include this label either: https://www.mozilla.org/en-US/firefox/developer/). 

There's no `email_label` passed to the[ newsletter form call in the code](https://github.com/mozilla/bedrock/blob/main/bedrock/firefox/templates/firefox/developer/index.html#L100). According to [helper definition](https://github.com/mozilla/bedrock/blob/67fd925b4d621dde0e48e27738c4b9c397ac5169/bedrock/newsletter/templatetags/helpers.py#L38), default should be `None`, so it seems like the `if not` condition should be true and `ftl('newsletter-form-your-email-address')` should be used. But I'm getting a bit lost on what to expect in this case. 

**suggestion**: might be out of scope for this fix, but it would be nice to have a test for conditional labels like this so it's clear when the label should show

Reply:
I don't think there's an issue with the conditional here. There's some CSS in the developer page that visually hides the email label. This was probably deliberate at the time for some design reason, but I'm going to call it a bug today and remove the CSS rule. Nice catch

Reply:
Ah, should have checked the inspector to rule out CSS! Thanks for the explanation